civictechprojects/static/css/partials/_PositionList.scss

```diff
@@ -0,0 +1,35 @@
+.PositionList-entry {
+  background-color: $color-background-light;
+  box-shadow: 0 3px 5px rgba(0,0,0,0.2);
+  padding: 5px;
+  margin: 15px;
+  max-width: 600px;
+  display: flex;
+  justify-content: space-between;
+  cursor: grab;
+
+  i {
+    cursor: pointer;
+  }
+  i:hover {
+    color:$color-orange;
+  }
+}
+
+i.dim {
+  color:$color-grey-disabled;
+}
+
+i.grip {
+  color:$color-grey-disabled;
+  padding-right: 5px;
+}
+i.grip:hover {
+  color:$color-grey-disabled;
+}
+
+.right-side {
+  display: flex;
+  justify-content: space-between;
+  width: 80px;
```

Comment:
Add `margin: auto 0;` to the right-side class and see if that fixes the vertical align issue?

Reply:
> 
> 
> Add `margin: auto 0;` to the right-side class and see if that fixes the vertical align issue?

That's a bit too specific, but 
```css
.left-side i, .right-side i {
margin: auto 0;
}
```

should do it.